test/screenshot/lib/controller.js

```diff
@@ -299,7 +307,7 @@ class Controller {
           console.log('\n\nDONE diffing screenshot images!\n\n');
           console.log(diffs);
           console.log(`\n\nFound ${diffs.length} screenshot diffs!\n\n`);
-          return {diffs, testCases};
+          return {testCases, diffs};
```

Comment:
Nit: not sure why this line changed?

Reply:
Just for consistency with the other `return` statements, to satisfy my OCD 😛